scss/_settings_font.scss

```diff
@@ -3,7 +3,7 @@ $font-base-family: '"Ubuntu", -apple-system, "Segoe UI", "Roboto", "Oxygen", "Ca
 $font-monospace: '"Ubuntu Mono", Consolas, Monaco, Courier, monospace' !default;
 $font-heading-family: $font-base-family !default;
 $font-use-subset-latin: false !default;
-$font-display-option: auto !default; // options - auto | block | swap | fallback | optional
+$font-display-option: optional !default; // options - auto | block | swap | fallback | optional
```

Comment:
@bartaz is the comment necessary?

Reply:
I don't know, it was there so I left it ;)
I guess it's fine to have it, so devs see what the values are if they want to override.